eq-author/src/App/QuestionnairesPage/QuestionnairesView/QuestionnairesTable/Row/__snapshots__/index.test.js.snap

```diff
@@ -45,6 +45,9 @@ exports[`Row should render 1`] = `
           date="2017/01/02"
         />
       </Row__TD>
+      <Row__TD>
+        <FormattedDate />
```

Comment:
Should this contain the updated date?

Reply:
Yeah you’re right good spot I've just pushed up a fix